typeof FormData error at build

Current version is: "mailgun.js": "^3.2.0",
Getting errors like below: 

```
node_modules/mailgun.js/dist/index.d.ts(5,27): error TS2749: 'FormData' refers to a value, but is being used as a type here. Did you mean 'typeof FormData'?
node_modules/mailgun.js/dist/lib/client.d.ts(28,45): error TS2304: Cannot find name 'FormData'.
node_modules/mailgun.js/dist/lib/request.d.ts(8,52): error TS2304: Cannot find name 'FormData'.
```

The API works fine on local but will not build on production due to error above. When can we expect updates for this?

I too am running into these three errors. I am attempting to replace mailgun-js with this officially-supported replacement.

```
"typescript": "^4.1.3"
"ts-node": "^9.1.1"
"form-data": "^4.0.0"
"mailgun.js": "^3.3.0"
```

It should be noted that the first of the three errors is only different because the `FormData` function argument is getting interpreted as the the value that it then tries to use as a type. Changing `constructor(FormData: new () => FormData);` to `constructor(formData: new () => FormData);` (note lowercase `f`) results in all three errors being the same:
> error TS2304: Cannot find name 'FormData'.

In other words, typescript is unable to find a FormData global.

## temporary workaround

A workaround I am using until it is resolved is to inject my own FormData type override. I have [typeRoots](https://www.typescriptlang.org/tsconfig#typeRoots) setup in my tsconfig, and **in the type root directory, I created a file `FormData/index.d.ts` with the contents**
```ts
type FormData = unknown;
```

Although hacky & sad, this allows my project to build without errors.

@andrii-yelis I'm also having the same issues outlined above, plus the errors:

```
This expression is not constructable.
  Type 'typeof import(".../node_modules/mailgun.js/dist/index")' has no construct signatures.
```

```
node_modules/mailgun.js/lib/error.ts:17:5 - error TS2322: Type 'null' is not assignable to type 'string'.

17     this.stack = null;
```

And many errors on `node_modules/ky/index.d.ts` about undefined names `Response`, `Request`, etc.

--- a/index.ts
+++ b/index.ts
@@ -4,7 +4,7 @@ import Options from './lib/interfaces/Options';
 export default class Mailgun {
   private formData: new () => FormData
 
-  constructor(FormData: new () => FormData) {
+  constructor(FormData: new (...args: any[]) => FormData) {
     this.formData = FormData;
   }
 

--- a/lib/client.ts
+++ b/lib/client.ts
@@ -30,7 +30,7 @@ export default class Client {
   public ips;
   public ip_pools;
 
-  constructor(options: Options, formData: new () => FormData) {
+  constructor(options: Options, formData: new (...args: any[]) => FormData) {
     const config: RequestOptions = { ...options } as RequestOptions;
 
     if (!config.url) {

